### PR TITLE
FOGL-3629: Fixed regex for replacing key value in validation expression (PATCH)

### PR DESCRIPTION
**Pull request by shariq1501**

### CODE REVIEW

**Review by AmandeepArora — DISMISSED** [review]
Works well in my setup.

File: src/app/components/core/configuration-manager/view-config-item/view-config-item.component.ts [hunk 1]
```diff
@@ -409,17 +409,25 @@ export class ViewConfigItemComponent implements OnInit, OnChanges, OnDestroy {
         });
       }
 
+
+
```
Comment (praveen-garg) [inline src/app/components/core/configuration-manager/view-config-item/view-config-item.component.ts]: Please remove this 2 extra newlines

File: src/app/components/core/configuration-manager/view-config-item/view-config-item.component.ts [hunk 2]
```diff
@@ -409,17 +409,25 @@ export class ViewConfigItemComponent implements OnInit, OnChanges, OnDestroy {
...
       for (const k in data) {
         data[k].key = k;
+
```
Comment (praveen-garg) [inline src/app/components/core/configuration-manager/view-config-item/view-config-item.component.ts]: remove this newline?

File: src/app/components/core/configuration-manager/view-config-item/view-config-item.component.ts [hunk 3]
```diff
@@ -409,17 +409,25 @@ export class ViewConfigItemComponent implements OnInit, OnChanges, OnDestroy {
...
+            if (regex.test(data[k].validityExpression)) {
+              data[k].validityExpression = data[k].validityExpression.replace(`${el.key}`, `"${el.value}"`);
+            }
           });
         }
       }
 
+      console.log('asset', data);
+
+
```
Comment (praveen-garg) [inline src/app/components/core/configuration-manager/view-config-item/view-config-item.component.ts]: remove these comments and extra newlines

**Review by praveen-garg — DISMISSED** [review]
let's do the cleanup and remove  comment. and do a squash merge. /cc @monika-Dianomic

**Review by praveen-garg — APPROVED** [review]

**Pull request merged**